src/serac/physics/utilities/functional/tests/functional_boundary_test.cpp

```diff
@@ -12,91 +12,201 @@
 #include "axom/slic/core/SimpleLogger.hpp"
 
 #include "serac/serac_config.hpp"
-#include "serac/numerics/mesh_utils.hpp"
+//#include "serac/numerics/mesh_utils.hpp"
```

Comment:
dead code.

Reply:
This can actually be replaced with `mesh_utils_base.hpp` so we don't have to copy/paste the `buildMesh` functions here